envoy_build_config/extensions_build_config.bzl

```diff
@@ -9,7 +9,7 @@ EXTENSIONS = {
     "envoy.filters.http.dynamic_forward_proxy":            "//source/extensions/filters/http/dynamic_forward_proxy:config",
     "envoy.filters.http.local_error":                      "@envoy_mobile//library/common/extensions/filters/http/local_error:config",
     "envoy.filters.http.platform_bridge":                  "@envoy_mobile//library/common/extensions/filters/http/platform_bridge:config",
-    "envoy.filters.http.socket_selection":                 "@envoy_mobile//library/common/extensions/filters/http/socket_selection:config",
+    "envoy.filters.http.network_configuration":                 "@envoy_mobile//library/common/extensions/filters/http/network_configuration:config",
```

Comment:
indentation is off here, no need to block and rerun CI for this though. Follow up?